package.json

```diff
@@ -53,5 +53,23 @@
   },
   "engines": {
     "node": "12.x"
-  }
+  },
+  "description": "<div>   <img alt=\"TeacherFund logo\" src=\"https://github.com/teacherfund/TeacherFund_next/raw/production/public/images/Logo.png\" width=\"200px\"> </div>",
+  "version": "1.0.0",
```

Comment:
this all needs to be removed

Reply:
sir, do you want me to remove description and version

Reply:
description and version are fine, thanks!